### ISSUE TITLE
Feat : display the attachment directly in the comment or news

### ISSUE THREAD

**Issue by agile-marmot** [tracim/tracim#5401]
## Feature description and goals

When one wishes to publish a document with a text, the text is displayed as a comment, which is neither intuitive nor useful. Often we want to share a text that is accompanied by a document and that explains it => The text written by the person should be displayed, and underneath simply the logo or small preview and the title of the attachment.

**Comment by alexgarel** [comment]
I would also propose this inside out (to enable commenting on a set of files): add the possibility to add attachments to a note. You can currently add an attachment in the comments, but not directly to the document. So one has to open discussion panel to see it (also discussion attachments seems more like transient data).

The attachments of a note could be displayed at the end of the notes.

I have built different documentation system with non technical user, and most of the time, the "attachment" feature really helps, it's far more easy to them than uploading files and then linking them in a document. They are already used to this pattern through emails.

To implement this:
* either we add a new attachments paradigm to the note,
* either we simply add a property to documents uploaded in the discussion to make them appear as attachment (with the possibility to detach it later, if needed)

**Comment by lebouquetin** [comment]
Hello,

Thanks for the suggestion. I had this feature in mind and the very first idea I add was to add a tab in right panel in which all attached files would appear. Is this what you suggest in your second option? I feel like what you suggest would be to also allow to "remove" history attached files from this tab.

Do you understand what I mean @alexgarel ? Let me know what you think about this.

**Comment by alexgarel** [comment]
@lebouquetin yes that's close enough at least. But maybe it should be quite visible how many attachments there are, to trigger a click on it.
